playbooks/roles/figures/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+---
+
+# This role is based on a temporary hack
+# This should configure figures in a prod instance but that's not the correct method
+
+- include_vars: ../../common_vars/defaults/main.yml
+- include_vars: ../../edxapp/defaults/main.yml
+
+- name: create figures parent and installation directories
+  file:
+    path: "{{ item.path }}"
+    state: directory
+    mode: 0755
+    owner: "edxapp"
+    group: "{{ common_web_group }}"
+  with_items:
+    - { path: "{{ figures_parent_dir }}" }
+    - { path: "{{ figures_code_dir }}" }
+  tags:
+    - figures
+
+- name: uninstall any previous versions of figures
+  command: "{{ edxapp_venv_dir }}/bin/pip uninstall -y {{ item }}"
+  become_user: "{{ edxapp_user }}"
+  environment: "{{ edxapp_environment }}"
+  register: figures_exists
+  ignore_errors: yes
+  with_items: 
+    - "figures"
+  tags:
+    - figures
+    - uninstall:figures
+
+# if figures existed before, chances are that the static assets were built
+# that would fail git checkout because local modifications will exists in the repo
+# reset HEAD before checking out
```

Comment:
@tkeemon so, the files are first generated within the repo and then you have to compile LMS assets for the LMS to pick the figures assets and serve them from within LMS. 
At least that's what I understand about this 😂 
But other than that, adding those files to .gitignore is not feasible because I think we don need to check those files in dev environment. 
So, this is our only option for now.